app/translations/cy/census_individual_gb_wls_cy.po

```diff
@@ -1579,12 +1583,12 @@ msgstr "Cymraeg neu Saesneg"
 
 #. answer-id: language-answer
 msgctxt "Answer for: What is <em>{person_name_possessive}</em> main language?"
-msgid "Other"
-msgstr "Arall"
+msgid "Other, including British Sign Language"
+msgstr "Arall, gan gynnwys laith Arwyddion Prydain"
```

Comment:
There is a typo here- the word 'laith' should be 'iaith/Iaith'.

Reply:
Good spot! Fixed.